README.md

```diff
@@ -67,6 +67,36 @@ python3 ./server.py 8880 &
 
 Then open <http://localhost:8880> in your browser.
 
+## Backporting Manual Changes
+
+In the event that a backported update is necessary, manual deployment of the newly built HTML is handled via Git tags, which are version-numbered tags based on major Lean versions matching the RegEx pattern `^v4\.\d+\.\d+$`. Pushing a new tag triggers the necessary deployment, but to do so requires first deleting any existing tag of the same name. The following describes the steps and commands you may use to manage this process.
+
+Make any desired changes to the codebase. Once you have tested changes locally, create a new branch based on the version of the reference manual you are modifying. By convention, this branch should match the tag (i.e. version number), but without the leading `v`. Push this branch to the remote.
+
+For example, for modifications to the reference manual associated with Lean version 4.22, the branch should be named `4.22.0`:
+
+```
+git checkout -b 4.22.0  //create new branch
```

Comment:
Here, we should add a step to check if the branch exists before just creating it. If there's already a maintenance branch for a given release, it should be used.

Reply:
It won't let you push if you don't have all the changes in it. Maybe this should be phrased in Git verbs rather than commands?